app/styles/globals.css

```diff
@@ -5,17 +5,22 @@
 
 html,
 body {
+  overflow: hidden;
```

Comment:
Dont think we need this? If we add it then our whole app wont be scrollable.  On the div with the house list, add `max-h-screen`